cxx-sensors/src/main/java/org/sonar/cxx/sensors/functioncomplexity/CxxFunctionComplexitySquidSensor.java

```diff
@@ -206,8 +204,7 @@ private double calculatePercentual(int overThreshold, int belowThreshold){
     double total = (double)overThreshold + (double)belowThreshold;
     if (total > 0) {
       return (overThreshold * 100.0) / ((double)overThreshold + (double)belowThreshold);
-    }
-    else {
+    } else {

```

Comment:
funciton `private double calculatePercentual(int overThreshold, int belowThreshold){` should be static